backendless.d.ts

```diff
@@ -833,6 +837,10 @@ declare module Backendless {
 
         getRelationName(): string;
 
+        setDistinct(distinct: boolean): this;
```

Comment:
the DataQueryBuilder already has these methods

Reply:
+